src/main/resources/connectionParameters.properties

```diff
@@ -6,6 +6,8 @@ cdfurl=http://localhost:11011/pipelines/ns/default/studio
 cdfConnectionsUrl=http://localhost:11011/cdap/ns/default/connections
 # CDAP Replication url
 replicationUrl=http://localhost:11011/cdap/ns/default/replication/create
+# CDF BYOID url
+cdfbyoidurl=https://test-byoid-cdf-url/
```

Comment:
Will this instance be in cdf-byoid project?

Reply:
This is a placeholder CDF URL text. It will be replaced by the Cloud build with the targeted CDF instance URL.